libs/design/loading-icon/examples/src/loading-icon-color copy/loading-icon-color.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
This whole file and `loading-icon-color copy` folder is not needed?

Reply:
eb818c7c52915d44a9afa366ed1ba85e54c1217f